### ISSUE TITLE
conda installation error

### ISSUE THREAD

**Issue by weiHelloWorld** [openpathsampling/openpathsampling#720]
Hi all,

I tried to install the package using conda following the instruction here: http://openpathsampling.org/latest/install.html, but I got following error information:

```
UnsatisfiableError: The following specifications were found to be in conflict:
  - navigator-updater
  - openpathsampling -> pyqt 4* -> qt 4.8.6
  - openpathsampling -> pyqt 4* -> sip 4.16.5
Use "conda info <package>" to see the dependencies for each package
```

By running `conda info openpathsampling`, I got following output:

```
openpathsampling 0.9.0 py27_0
-----------------------------
file name   : openpathsampling-0.9.0-py27_0.tar.bz2
name        : openpathsampling
version     : 0.9.0
build string: py27_0
build number: 0
channel     : omnia
size        : 547 KB
arch        : x86_64
has_prefix  : False
license     : LGPL 2.1 or later
md5         : d8c2cf67b1897744ffcbf4dd115b8e15
noarch      : None
platform    : linux
requires    : ()
subdir      : linux-64
url         : https://conda.anaconda.org/omnia/linux-64/openpathsampling-0.9.0-py27_0.tar.bz2
dependencies:
    matplotlib
    mdtraj
    netcdf4
    networkx
    numpy
    openmm
    openmmtools
    pandas
    pyqt 4*
    python 2.7*
    pyyaml
    scipy
    svgwrite

openpathsampling 0.9.1 py27_0
-----------------------------
file name   : openpathsampling-0.9.1-py27_0.tar.bz2
name        : openpathsampling
version     : 0.9.1
build string: py27_0
build number: 0
channel     : omnia
size        : 551 KB
arch        : x86_64
has_prefix  : False
license     : LGPL 2.1 or later
md5         : dae126b286ef3431fae5bb5b2a3c9562
noarch      : None
platform    : linux
requires    : ()
subdir      : linux-64
url         : https://conda.anaconda.org/omnia/linux-64/openpathsampling-0.9.1-py27_0.tar.bz2
dependencies:
    matplotlib
    mdtraj
    netcdf4
    networkx
    numpy
    openmm
    openmmtools
    pandas
    pyqt 4*
    python 2.7*
    pyyaml
    scipy
    svgwrite
    ujson

openpathsampling 0.9.2 py27_0
-----------------------------
file name   : openpathsampling-0.9.2-py27_0.tar.bz2
name        : openpathsampling
version     : 0.9.2
build string: py27_0
build number: 0
channel     : omnia
size        : 709 KB
arch        : x86_64
has_prefix  : False
license     : LGPL 2.1 or later
md5         : 8c7207396fd021afdd390489445c4f49
noarch      : None
platform    : linux
requires    : ()
subdir      : linux-64
url         : https://conda.anaconda.org/omnia/linux-64/openpathsampling-0.9.2-py27_0.tar.bz2
dependencies:
    jupyter
    matplotlib
    mdtraj
    netcdf4
    networkx
    numpy
    openmm
    openmmtools
    pandas
    pyqt 4.11.4 py27_3
    python 2.7*
    pyyaml
    scipy
    svgwrite
    ujson
```

Could anyone tell me what the problem is and how should I fix it?

Thanks!

**Comment by dwhswenson** [comment]
It looks to me like there are problems based on our old release (0.9.2, released 1 Dec 2016). The conda recipe for that release included specific pins to requirement versions, based on problems that have since been resolved. I think those pins are causing the problems you see.

We're very close to a new release (see relevant issue here: https://github.com/omnia-md/conda-recipes/pull/816), so if installation isn't urgent, you can watch there for the conda-installable version to be ready. 

Or, you can try the following procedure:

1.  Manually install the requirements listed in the `requirements.run` section of our [`devtools/conda-recipe/meta.yaml`](https://github.com/openpathsampling/openpathsampling/blob/master/devtools/conda-recipe/meta.yaml) (all of which should be conda-installable, either in the `omnia` or `conda-forge` channels)
2. Download the current GitHub version (or the [release at 0.9.3](https://github.com/openpathsampling/openpathsampling/releases/tag/v0.9.3))
3. Install in the usual way with `python setup.py install`.

This should work. If you do that, I recommend also installing `nose` (via conda) and running the tests with `nosetests -v` (from within the GitHub-downloaded directory) to ensure that everything is working.

Keep in mind that OPS is still beta code (although we're already using it in our own production projects), so please keep an eye out here for links to the papers documenting the code once they get published. Let us know if this fixes you problems, and if any other problems come up!

**Comment by weiHelloWorld** [comment]
OK, it works, thank you!

**Comment by dwhswenson** [comment]
Glad that got it working. Just as an update to anyone who comes across this, the conda-recipes PR I mentioned was just merged, so 0.9.3 should be up and running in the omnia channel of conda.
